Source/SwiftLintFramework/Rules/ProhibitedSuperRule.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  ProhibitedSuperRule.swift
+//  SwiftLint
+//
+//  Created by Aaron McTavish on 12/12/16.
+//  Copyright © 2016 Realm. All rights reserved.
+//
+
+import SourceKittenFramework
+
+public struct ProhibitedSuperRule: ConfigurationProviderRule, ASTRule, OptInRule {
```

Comment:
why should this be an opt-in rule? seems like it should be enabled by default...